Documentation/policy/language.rst

```diff
@@ -487,25 +500,104 @@ Example
 
         .. literalinclude:: ../../examples/policies/l3/fqdn/fqdn.json
 
-Limitations
-~~~~~~~~~~~
 
-The current ``toFQDNs`` implementation is very limited. It may not behave as expected.
+Managing Long-Lived Connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Often, an application may keep a connection open for longer than the configured
+DNS TTL. Without further DNS queries the remote IP used in the long-lived
+connection may expire out of the DNS cache. When this occurs, existing
+connections will become disallowed by policy and will be blocked. In cases
+where an application retries the connection, a new DNS query is issued and the
+IP is added to the policy.
+
+A minimum TTL is used to ensure a lower bound to DNS data expiration, and DNS
+data in the Cilium DNS cache will not expire sooner than this minimum. It
+can be configured with the ``--tofqdns-min-ttl`` CLI option. The value is in
+integer seconds and must be 1 or more. The default is 1 hour.
+
+.. note:: It is recommended that ``--tofqdns-min-ttl`` be set to the minimum
+          time a connection must be maintained.
+
+
+Obtaining DNS Data
+~~~~~~~~~~~~~~~~~~
+IPs are obtained via DNS polling or intercepting DNS requests via a proxy, and
+matching names are inserted irrespective of how the data is obtained. These IPs
+can be selected with ``toFQDN`` rules. DNS responses are cached within cilium
+agent respecting TTL.  
+
+.. _DNS Proxy:
+
+DNS Proxy
+  A DNS Proxy intercepts DNS traffic and records IPs seen in the responses.
+  This interception is itself a separate policy rule governing the DNS
+  requests, and must be specified separately. For details on how to enforce
+  policy on DNS requests and configuring the DNS proxy, see `Layer 7 Examples`_.
+
+  Intercepting DNS requests allows Cilium to generate the IPs allowed by policy
+  from the IPs received in DNS responses to applications. It is also required
```

Comment:
This should mention that this allows maintaining the IP mapping for individual endpoints and not per node.